homeassistant/components/life360/device_tracker.py

```diff
@@ -166,7 +166,11 @@ def _handle_coordinator_update(self) -> None:
 
     @property
     def force_update(self) -> bool:
-        """Return True if state updates should be forced."""
+        """Return True if state updates should be forced.
+
+        Overridden because CoordinatorEntity sets should_poll to False,
+        which causes TrackerEntity to default to True.
```

Comment:
Instead of this, should we change `TrackerEntity` or add a `CoordinatorTrackerEntity`?

Reply:
AFAICT, this is the only `device_tracker` component that has overridden `force_update`
I am not sure if it warrants adjusting in the base class.

Reply:
Sorry, I'm wondering, why can't this integration use `_attr_force_update` (instead of the overridden method) like all the others? I'm missing why adding `_attr_force_update = False` wouldn't work here.